frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.dedicated-resources.scenario.ts

```diff
@@ -0,0 +1,54 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { withResource, click } from '@console/shared/src/test-utils/utils';
+import * as editDedicatedResourcesView from '../views/editDedicatedResourcesView';
+import * as virtualMachineView from '../views/virtualMachine.view';
+import { VM_CREATE_AND_EDIT_TIMEOUT_SECS } from './utils/consts';
+import { VirtualMachine } from './models/virtualMachine';
+import { vmConfig, getProvisionConfigs } from './vm.wizard.configs';
+import { ProvisionConfigName } from './utils/constants/wizard';
+
+describe('KubeVirt VM detail - edit Dedicated Resources', () => {
+  const leakedResources = new Set<string>();
+  const provisionConfigs = getProvisionConfigs();
+
+  const configName = ProvisionConfigName.CONTAINER;
+  const provisionConfig = provisionConfigs.get(configName);
+
+  provisionConfig.networkResources = [];
+  provisionConfig.storageResources = [];
+
+  it(
+    'enables dedicated resources guaranteed policy, then disables it',
+    async () => {
+      const vm1Config = vmConfig(configName.toLowerCase(), testName, provisionConfig);
+      vm1Config.startOnCreation = false;
+
+      const vm = new VirtualMachine(vmConfig(configName.toLowerCase(), testName, provisionConfig));
+      await withResource(leakedResources, vm.asResource(), async () => {
+        await vm.create(vm1Config);
```

Comment:
We probably don't need to create the VM with Wizard, since wizard is not beign tested here. You can take a look at for example `packages/kubevirt-plugin/integration-tests/tests/vm.detail.boot-order.scenario.ts` to see how a VM is created with `createResource` method.
It's faster than creating the VM with wizard

Reply:
it fails for me, `boot-order.scenario` fails aswell..can you check with your env @rhrazdil ?